features/configuration/read_options_from_file.feature

```diff
@@ -16,6 +16,24 @@ Feature: read command line configuration options from files
   lowest to highest precedence; for example, an option in `~/.rspec` can be
   overridden by an option in `.rspec-local`).
 
+  **Windows users:** The output uses ANSI escapte codes to show text in color.
+  If you see ANSI escape codes in the output (something like `[1m [31m` ) and
+  your text isn't in different colors, you may need to install a utility so
+  that your Windows shell will interpret those codes correctly and show the
+  colors.  Here are some popular solutions:
+
+    * [ANSICON](https://github.com/adoxa/ansicon): ANSICON runs 'on top of' cmd or powershell. This is a very
+      popular solution. You can set it up so that it's always used whenever
+      you use cmd or powershell, or use it only at specific times.
+
+    * Alternatives to cmd.exe or powershell: [ConEmu](http://conemu.github.io/), [Console2](http://sourceforge.net/projects/console/),
+      [ConsoleZ](https://github.com/cbucher/console)
+
+    * Unix-like shells and utilities:  [cygwin](https://www.cygwin.com/), [babun](http://babun.github.io/index.html),
+      [MinGW](http://www.mingw.org/) (Minimalist GNU for Windows)
+
+  More information can be found by searching for information about those solutions.
+
```

Comment:
That makes good sense to me.  That will be much more obvious on [Relish](http://www.relishapp.com/rspec/rspec-core/docs) and anywhere else.  I'll make an entry in ```.nav```.  I'll put it after ```- read_options_from_file.feature``` because that references color and also because if people are having trouble with color, they'll see it quickly.  But of course I'm happy to put it where-ever you think it should go in ```.nav```.